src/py42/clients/settings/converters.py

```diff
@@ -0,0 +1,48 @@
+from py42._compat import str
+
+
+def bool_to_str(value):
+    if isinstance(value, bool) or value in ("true", "false"):
+        return str(value).lower()
+    else:
+        raise ValueError("Value must be True or False")
```

Comment:
So the goal with these converters is to smooth out the oddities of the settings APIs. In this case the fact that the required json values need to be the strings `"true"` and `"false"` instead of json booleans. We don't want to have end-users of py42 have to remember which values need to be strings and which need to be booleans, so we do the conversion for them, so they can just do: 

```
if org_setting.some_property:
     org_setting.some_property = False
```

But if they _do_ happen to assign the string value, we just use it. 

And for why this one raises an AttributeError, is because it's being used as a `set_converter` on many `SettingProperty` classes. The ones that don't raise AttributeError are only being used as `get_converter`s to transform the current values (which we expect to never be invalid since we're getting them from the server). 

Basically, the goal is to make sure the end user doesn't have to know the ins and outs of the underlying nested settings dict and all it's quirks, just that each "setting" that's available in the console UI has a corresponding property (or sometimes method) on the `OrgSettings`/`DeviceSettings` object. The property classes then do the work to convert values _from_ the underlying dicts to something user-friendly, as well as convert the user-friendly representation back to what the settings dicts require.

Reply:
Right, @kiran-chaudhary `True` just means the boolean here, not the string